# Tailscale-Version: 1.102.2
# Description: Change subnet route fwmark from 0x40000 to 0x800000 to avoid conflict with UniFi wgclt1
--- a/tsconst/linuxfw.go
+++ b/tsconst/linuxfw.go
@@ -33,8 +33,8 @@
 
 	// Packet is from Tailscale and to a subnet route destination, so
 	// is allowed to be routed through this machine.
-	LinuxSubnetRouteMark    = "0x40000"
-	LinuxSubnetRouteMarkNum = 0x40000
+	LinuxSubnetRouteMark    = "0x800000"
+	LinuxSubnetRouteMarkNum = 0x800000
 
 	// Packet was originated by tailscaled itself, and must not be
 	// routed over the Tailscale network.
